### PR TITLE
Added cvelascof to 'contributors.json'.

### PR DESCRIPTION
**Pull request by pp-mo**

### CODE REVIEW

File: contributors.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
 { "profile_name":"SimonPeatman", "organisation":"University of Reading"},
 { "profile_name":"pelson", "organisation":""},
 { "profile_name":"malcolmbrooks", "organisation":""}
+{ "profile_name":"cvelascof", "organisation":""},
```
Comment (ajdawson) [inline contributors.json]: This is not valid JSON and will cause the about page not to render. You need a comma after every entry in a container **except** the last. Here you have no comma after the second-to-last *and* a comma after the last.

Comment (pp-mo) [inline contributors.json]: Whoops.
You respond so quick, I was only just reviewing it myself @ajdawson !

**Comment by ajdawson** [comment]
Looks fine. A quick way to test is to run an HTTP server and browse the site locally, checking that the about page renders as expected:

    python -m SimpleHTTPServer

for Python2, and for Python 3:

    python -m http.server

in the root directory of this repository.

**Comment by pp-mo** [comment]
> A quick way to test
I really never thought to do that !
Thanks @ajdawson .

**Pull request merged**